content/03.results.md

```diff
@@ -11,11 +11,11 @@ We demonstrate ... / summarize results or something here
 Pangenomes from isolates are typically built by assembling each isolate genome and predicting genes (open reading frames), clustering gene sequences from all genomes into a non-redundant set, and estimating the presence/absence or abundance of each gene in each genome. 
 To determine whether bacterial and archaeal pangenomes could be constructed from reduced alphabet k-mers, we compared pangenomes estimated from genes against those estimated from k-mers (amino acid, dayhoff, and hydrophobic-polar).
 We compared pangenomes from 23 species belonging to 23 phyla in the GTDB taxonomy (CITE), with pangenome size ranging from 20-972 genomes (mean = 203 genomes).
-For each pangenome, we compared the total number of genes and k-mers and the total number of unique genes and k-mers within each genome. 
+For each pangenome, we compared the total number of genes, number of unique genes, total k-mers, and number of distinct k-mers within each genome.
 We also tested the similarity of presence/absence profiles between pangenomes constructed with different methods using the Mantel test.
 
 For these three metrics, performance varied minimally across encodings and k-mer sizes (**Figure @fig:violin_fig**).
-This is likely because the genomes of the same species are closely related, so any reduced alphabet k-mer is sufficient to overcome minor variations such as those introduced by third base pair wobble (CITE).
+This is likely because the genomes of the same species are closely related, so any reduced alphabet k-mer is sufficient to overcome minor variations such as those introduced by codon degeneracy or evolutionary drift (CITE).
```

Comment:
> but I am actually thinking that we should remove codon degeneracy because all three of these encodings explicitly ignore codon degeneracy!

I think I like this update as-is...all three encodings "ignore" codon degeneracy in that degenerate codons would be masked, which does allow these encodings to overcome minor nucleotide variations introduced by synonymous mutations.

Reply:
what if we updated the text to read "minor genomic variations" instead of just "minor variations"?

Reply:
+1